.changeset/silly-eels-admire.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+Added the possibility to authorize actions
+
+It is now possible to decide who should be able to execute certain actions or who should be able to pass specific input to specified actions.
```

Comment:
Think you should also callout the renames that you've made even if it is for the alpha types.